src/VisualStudio/Core/Next/project.json

```diff
@@ -21,6 +21,7 @@
     "RoslynDependencies.Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime": "15.0.25718-Preview5",
     "RoslynDependencies.Microsoft.VisualStudio.Workspace": "14.0.983-pre-ge167e81694",
     "System.Collections.Immutable": "1.3.1",
+    "System.Threading.Tasks.Dataflow": "4.5.24",
```

Comment:
This used to get resolved via the GAC.  Even though it's not directly referenced it's an item that MSBuild inserts as a needed reference due to some other dependencies.  GAC used to be saving us here.  Now we need to be explicit.